test/com/google/enterprise/adaptor/opentext/OpentextDocIdTest.java

```diff
@@ -63,6 +63,15 @@ public void testInvalidIdMissingNodeId() {
     OpentextDocId docId = new OpentextDocId(adaptorDocId);
   }
 
+  @Test
+  public void testInvalidIdNonNumeric() {
+    thrown.expect(IllegalArgumentException.class);
+    thrown.expectMessage("InvalidId:foo");
+
+    DocId adaptorDocId = new DocId("InvalidId:foo");
```

Comment:
Which line throws the exception? If the first, the second is dead code. If the second, the expects should be moved down.